src/MSBuild/LiveLogger/LiveLogger.cs

```diff
@@ -377,8 +377,20 @@ private void ProjectFinished(object sender, ProjectFinishedEventArgs e)
                             // Ignore any GetDirectoryName exceptions.
                         }
 
+                        string urlString;
+                        try
+                        {
+                            // This should generate file:// schema url string which is better handled by various Terminal clients than raw folder name.
+                            urlString = new Uri(url.ToString()).AbsoluteUri;
+                        }
+                        catch
+                        {
+                            // If Uri constructor throws use raw folder name instead.
+                            urlString = url.ToString();
+                        }
```

Comment:
Suggestion: Use [TryCreate](https://learn.microsoft.com/en-us/dotnet/api/system.uri.trycreate?view=net-8.0) instead of try/catch?